### PR TITLE
fix: Collection pagination

### LINKED ISSUE #2626: Collection pagination
#### Steps to reproduce
- go to any collection (tested on `/rmrk/collection/800f8a914281765a7d-KITTY`)
- click on page 3
- click on any nft
- click on go back to previous page
- you're now on page 1 while you have page 3 in url

#### Expected Behavior
I should be on page 3 instead of page 1

#### video
https://user-images.githubusercontent.com/9987732/159443990-d0c98a5b-6e07-4226-9c6f-cbaa572da9f5.mov

### PR DESCRIPTION
**Pull request by Jarsen136**
**Thank you for your contribution** to the [KodaDot NFT gallery](https://kodadot.xyz).
👇 \_ Let's make a quick check before the contribution.

### PR type

- [x] Bugfix
- [ ] Feature
- [ ] Refactoring

### What's new?

- [x] PR closes #2626 
- [ ] <brief_description_of_what_I've_added>

### Before submitting Pull Request, please make sure:

- [x] My contribution builds **clean without any errors or warnings**
- [x] I've merged recent default branch -- **main** and I've no conflicts
- [x] I've tried to respect high code quality standards
- [x] I've didn't break any original functionality
- [x] I've posted a screenshot of demonstrated change in this PR

### Optional

- [x] I've tested it at </rmrk/collection/26902bc2f7c20c546a-1FVG7>
- [x] I've tested PR on mobile and everything seems works
- [x] I found edge cases
- [ ] I've written some unit tests 🧪

### Had issue bounty label?

- [x] Fill up your KSM address: [Payout](https://beta.kodadot.xyz/transfer/?target=Caiv9TbPz68q5dC8EcHu5xKYPRnremimGzqmEejDFNpWWLG)

### Community participation

- [x] [Are you at KodaDot Discord?](https://discord.gg/35hzy2dXXh)

### Screenshot

- [ ] My fix has changed **something** on UI; a screenshot is best to understand changes for others.

### CODE REVIEW

File: components/rmrk/Gallery/CollectionItem.vue [hunk 1]
```diff
@@ -170,7 +170,7 @@ export default class CollectionItem extends mixins(
     listed: false,
   }
   public activeTab = 'collection'
-  private currentValue = 1
+  private currentValue = parseInt((this.$route.query?.page as string) || '1')
```
Comment (roiLeo) [inline components/rmrk/Gallery/CollectionItem.vue:173]: same as
```suggestion
  private currentValue = parseInt(this.$route.query?.page as string) || 1
```

**Review by roiLeo — APPROVED** [review]
lgtm

File: components/rmrk/Gallery/CollectionItem.vue (hunk 1 above)
Comment (Jarsen136) [reply components/rmrk/Gallery/CollectionItem.vue:173]: Alright

**Comment by yangwao** [comment]
pay 100 usd

**Comment by yangwao** [comment]
😍 Perfect, I’ve sent the payout 
💵 $100 @ 165.45 USD/KSM ~ 0.604 $KSM
🧗 [Caiv9TbPz68q5dC8EcHu5xKYPRnremimGzqmEejDFNpWWLG](https://kusama.subscan.io/account/Caiv9TbPz68q5dC8EcHu5xKYPRnremimGzqmEejDFNpWWLG)
🔗 [0xf6bb9234fe3d13b700e18048062eb49a34aa19483b20e683e2eaa860ac64113d](https://kusama.subscan.io/extrinsic/0xf6bb9234fe3d13b700e18048062eb49a34aa19483b20e683e2eaa860ac64113d)

🪅 Let’s grab another issue and get rewarded! 
🪄 [github.com/kodadot/nft-gallery/issues](https://github.com/kodadot/nft-gallery/issues/)

**Pull request merged**